### ISSUE TITLE
Windows notifications

### ISSUE THREAD

**Issue by thijsdamen** [pajlada/TempestNotifier#5]
I think it would be usefull to use the windows notification system for tempests.Combined with some settings.

e.g.
Tempest to notify: corruption, 20 rogue exiles
minimum upvotes: 2
maximum map level: 74

if a tempest is found with these parameters, a notification (with some internal cooldown) is send to the windows system notifying the user of the available tempest.

I woudn't mind trying to implement this myself, just want tomake sure there is atleast some interest and no major objections to this.

**Comment by pajlada** [comment]
Exactly, this is a planned feature. What I had in mind did not include any sort of customization, but it could be very useful. :+1:
